compiler/damlc/tests/BUILD.bazel

```diff
@@ -1012,6 +1012,6 @@ daml_test(
     name = "interface-implementation",
     srcs = glob(["interfaces-data-deps/interface-implementation/**/*.daml"]),
     data_deps = [":interface-definition.dar"],
-    deps = ["//daml-script/daml:daml-script-1.dev.dar"],
     target = "1.dev",
+    deps = ["//daml-script/daml:daml-script-1.dev.dar"],
```

Comment:
side question, what's the logic behind the bazel formatter? I thought it was "keep fields sorted but keep things like 'name', 'srcs' and 'deps' at the top" but this threw me off, and the `buildifier` output only says that the file must be reformatted, not why